File: dev/ClientApp/src/app/models/saman/tokenInfo.ts
export default class tokenInfo {
  constructor(
    public terminalId: number,
    public resNum: number,
    public status: number,
    public errorCode: number,
    public errorDesc: string,
    public token: string,
    public type: string,
    public responseCode: string
  ) { }
}

File: Models/saman/TokenInfo.cs
namespace G_IPG_API.Models;

public class TokenInfo
{
    public long TerminalId { get; set; }
    public long ResNum { get; set; }
    public int Status { get; set; }
    public int ErrorCode { get; set; }
    public string ErrorDesc { get; set; }
    public string Token { get; set; }
    public string Type { get; set; }
}